scripts/inject-cli-as-overload.py

```diff
@@ -278,6 +278,16 @@ def fill_implementation_stub(
         overload_fn='Client',
         class_method=False,
     ),
+    dict(
+        cli_entrypoint='gateway',
+        doc_str_title='Configure the Gateway inside a Flow. The Gateway exposes your Flow logic as a service to the internet according to the protocol and configuration you choose.',
+        doc_str_return='the new Flow object',
```

Comment:
are we sure about this?

Reply:
why not ? to me it makes sense because we do it for `Flow.add` and `Flow.__init__` : 
* args come from cli
* need to be maintained with a bot

Reply:
doc_str_return I mean

Reply:
Yes sure about this because we want to be able to chain methods